methods/getCacheKey.js

```diff
@@ -1,3 +1,17 @@
-module.exports = function(url) {
-  return `${this.settings.app.redis.prefix}-${url}`;
+module.exports = function(path, query) {
+  if (!query) {
+    return `${this.settings.app.redis.prefix}-${path}`;
+  }
+  let querystring = '';
+  const keys = Object.keys(query).sort();
+  let first = true;
+  keys.forEach((key) => {
```

Comment:
a better approach would be to do a keys.map to get into an array, then do ${path}?${querystring.join('&')}